index.html

```diff
@@ -56,29 +51,31 @@
       </p>
       <img class="Ellipse12" src="/images/Ellipse12.png" alt="shapes"/>
       <img class="Ellipse13" src="/images/Ellipse13.png" alt="shapes"/>
-      <p class="socialmediatag">LET'S CONNECT</p>
-      <section>
+      <div>
+        <p class="socialmediatag">LET'S CONNECT</p>
+      </div>
+      <div>
         <ul class="socialmediaicons">
           <li>
-            <a href="#"><i class="fa-brands fa-twitter fa-2x"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter fa-xl iconspace"></i></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-linkedin-in fa-2x"></i></a>
+            <a href="#"><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-medium fa-2x"></i></a>
+            <a href="#"><i class="fab fa-medium fa-xl iconspace iconspace"></i></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-github fa-2x"></i></a>
+            <a href="#"><i class="fa-brands fa-github fa-xl iconspace "></i></a>
           </li>
           <li>
-            <a href="#"><i class="fab fa-angellist fa-2x"></i></a>
+            <a href="#"><i class="fab fa-angellist fa-xl iconspace"></i></a>
           </li>
         </ul>
-      </section>
+        </div>
       <img class="Ellipse15" src="/images/Ellipse15.png" alt="shapes"/>
       <img class="Ellipse4" src="/images/Ellipse4.png" alt="shapes"/>
       <img class="rectangle5" src="/images/Rectangle5.png" alt="shapes"/>
```

Comment:
- [ ]  Same comment for the shapes here, please remove all shapes from your HTML and use a single background that has all the shapes as I showed you above! 😌